pyke/kepfold.py

```diff
@@ -108,6 +108,19 @@ def kepfold(infile, period, bjd0, outfile=None, bindata=False,
 
     if outfile is None:
         outfile = infile.split('.')[0] + "-{}.fits".format(__all__[0])
+    # check if there is period or BJD0 information
+    if np.any([(period is None),(bjd0 is None)]):
+        # open input file
+        instr = pyfits.open(infile, 'readonly')
+        try:
+            period = instr[3].header['PERIOD']
+            bjd0 = instr[3].header['BJD0']
+            kepmsg.log(logfile,'Using period and BJD0 from KEPBLS',verbose)
+        except:
+            errmsg = ('ERROR -- KEPFOLD: No period information found. Either specify \
+            period and BJD0 or run KEPBLS.'
+                      .format(outfile))
```

Comment:
No need to break this line. Also, ``outfile`` won't be included in the string unless you add ``{}`` somewhere.

Reply:
Hahaha I nicked the error message from below, missed that, thanks